### PR TITLE
Pagination for find_people at gaps of 50

### PR DESCRIPTION
**Pull request by thezillion**
### Issue Reference
This PR addresses the Issue : Fixes #

### Summarize
Since number of entries in Person finder is large, a gap of 50 would be good enough for /find_people.

screenshot from 2018-08-19 21-31-54

### CODE REVIEW

**Comment by akshayjoyinfo** [comment]
testing PR

**Comment by realslimshanky** [comment]
@thezillion please reference the issue you are trying to solve.

**Comment by realslimshanky** [comment]
@thezillion I can see that the deployed version already has pagination at 50 results. Check it out `https://keralarescue.in/find_people`

**Comment by biswaz** [comment]
ping me on slack after testing

**Comment by thezillion** [comment]
@realslimshanky I meant keeping the number of pagination buttons at the bottom at a gap of 50. Right now  everypage is shown.

File: mainapp/templates/mainapp/request_form.html [hunk 1]
```diff
@@ -128,6 +129,7 @@ <h3 id="myModalLabel">Allow Location - നിങ്ങളുടെ സ്ഥല
 		$('#instructionModal').on('hidden.bs.modal', function (e) {
 		  navigator.geolocation.getCurrentPosition(success, error, options);
 		})
+<<<<<<< HEAD
```
Comment (dennismphil) [inline mainapp/templates/mainapp/request_form.html]: Here as well

File: mainapp/templates/mainapp/add_person.html [hunk 2]
```diff
@@ -46,6 +46,9 @@ <h3>{{ message }}</h3>
   </form> 
 {% endblock %}
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (dennismphil) [inline mainapp/templates/mainapp/add_person.html]: Remove the merge conflicts

File: templates/base.html [hunk 3]
```diff
@@ -27,8 +27,12 @@
   {% bootstrap_css %}
 
   <link rel="stylesheet" type="text/css" href="{% static 'css/style.css' %}"/>
+<<<<<<< HEAD
```
Comment (dennismphil) [inline templates/base.html]: here also

**Comment by thezillion** [comment]
All conflicts have been resolved

File: mainapp/templates/mainapp/camp_requirements.html [hunk 4]
```diff
@@ -40,8 +40,11 @@ <h3>{{ message }}</h3>
 
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
   <script>
-  $('#id_gender_0').parents('.radio').hide();
-  $('#id_gender > .radio').removeClass('radio').addClass('radio-inline');
+  window.addEventListener('load', function() {
+    $('#id_gender_0').parents('.radio').hide();
```
Comment (biswaz) [inline mainapp/templates/mainapp/camp_requirements.html]: This page do not have a gender field. Can you remove these 2 lines

**Pull request merged**